constructor/construct.py

```diff
@@ -83,11 +83,18 @@
 is contained as a result of resolving the specs for `python 2.7`.
 '''),
 
-    ('menu_packages',          False, list, '''
+    ('menu_packages',           False, list, '''
 A list of packages with menu items to be installed. The packages must have
-necessary metadata in `Menu/<package name>.json`). Menu items are currently
-only supported on Windows. By default, all menu items will be installed;
-supplying this list allows a subset to be selected instead.
+necessary metadata in `Menu/<package name>.json`). By default, all menu items
+found in the installation will be created; supplying this list allows a
+subset to be selected instead. If an empty list is supplied, no shortcuts will
+be created.
```

Comment:
So, to clarify: if I want to create shortcuts on Windows only, I add the following to my `construct.yaml`

```
menu_packages: []  # [not win]
```